packages/plugins/rubygem-foreman_snapshot_management/rubygem-foreman_snapshot_management.spec

```diff
@@ -18,19 +18,54 @@ Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
 Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby >= 2.5
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: foreman-assets >= %{foreman_min_version}
 BuildRequires: foreman-plugin >= %{foreman_min_version}
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}ruby >= 2.5
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 Provides: foreman-plugin-%{plugin_name} = %{version}
 # end specfile generated dependencies
 
+# start package.json devDependencies BuildRequires
+BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
+BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 9.0.0
+BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) >= 3.0.0
+BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(lodash) >= 4.17.11
+BuildRequires: %{?scl_prefix}npm(lodash) < 5.0.0
+BuildRequires: %{?scl_prefix}npm(sortabular) >= 1.5.1
+BuildRequires: %{?scl_prefix}npm(sortabular) < 2.0.0
+BuildRequires: %{?scl_prefix}npm(table-resolver) >= 3.2.0
+BuildRequires: %{?scl_prefix}npm(table-resolver) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(react-intl) >= 2.8.0
+BuildRequires: %{?scl_prefix}npm(react-intl) < 3.0.0
+BuildRequires: %{?scl_prefix}npm(react-redux) >= 5.0.7
+BuildRequires: %{?scl_prefix}npm(react-redux) < 6.0.0
+BuildRequires: %{?scl_prefix}npm(redux) >= 3.7.2
+BuildRequires: %{?scl_prefix}npm(redux) < 4.0.0
+BuildRequires: %{?scl_prefix}npm(redux-thunk) >= 2.3.0
+BuildRequires: %{?scl_prefix}npm(redux-thunk) < 3.0.0
```

Comment:
Most of these are normally provided by `@theforeman/vendor`. Please reach out to the UI team to see if you can use that instead. Otherwise you risk loading dependencies twice (also at runtime) or other incompatibilities.

Reply:
I am still going to merge this as-is -- using the exported ones from `vendor` requires slightly different code anyways, so it would be a new release.